README.md

```diff
@@ -1,35 +1,59 @@
-# :fishing_pole_and_fish: config [![GoDoc][doc-img]][doc] [![GitHub release][release-img]][release] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![Report Card][report-card-img]][report-card]
+# :fishing_pole_and_fish: config [![GoDoc][doc-img]][doc] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov]
 
-Package `config` allows users to:
-
-* Get components working with minimal configuration
-* Override any field if the default doesn't make sense for their use case
+Convenient, injection-friendly YAML configuration.
 
 ## Installation
-We recommend locking to [SemVer](http://semver.org/) range `^1` using
-[Glide](https://github.com/Masterminds/glide):
 
 ```
-glide get 'go.uber.org/config#^1
+go get -u go.uber.org/config
 ```
 
-## Stability
+Note that config only supports the two most recent minor versions of Go.
+
+## Quick Start
+
+```
```

Comment:
let's add `golang` as the language to get coloured syntax

Reply:
Good point.